### ISSUE TITLE
Support for aarch64 and riscv

### ISSUE THREAD

**Issue by hyperswine** [rust-osdev/bootimage#83]
Would be nice to have minimal support for aarch64 and riscv bare metal targets.

**Comment by phil-opp** [comment]
Agreed! Unfortunately, I don't have time to look into this myself right now, but I'd be happy to accept PRs for this. See also https://github.com/rust-osdev/bootloader/issues/299.
